tools/unused.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+import os
+import re
+import b
+
+def register(command):
+    # Install help
+    command.help = "Creates a list of unused or commented out modules"
+
+
+def run(**kwargs):
+
+    source_dir = b.cmake_cache[b.cmake_cache["CMAKE_PROJECT_NAME"] + "_SOURCE_DIR"]
```

Comment:
Using `'` instead of `"` to be consistent with the rest of the files / other python files.

Reply:
Our auto-formatting forces `"` now